### ISSUE TITLE
Object not displayed properly in output when initialize assign something to "this"

### ISSUE THREAD

**Issue by EricTherrien** [jneale/Xplore#40]
Say the following code:

```
var x = Class.create();
x.prototype = {
  initialize: function( a ) {
   //this._a = a; 
  },
  test1 : function() { return 1; },
  test2 : "a"
};
y = new x();
y;
```

If you run this in Xplore (or if it's in a Script Include and invoked in Xplore), you get the expected result of listing the 2 functions and the "property" in the output.
If you un-comment the code in the initialize function on line 4, and run it, now you only get the _a "property" in the output.
It's only a display issue, as if you display the function or the "property" specifically (say end with y.test1) you do get the expected result. For some reason, when initialize assign something to "this", Xplore's output "forget" about everything else in the object.
This is happening with version 4.8.1, it was working as expected in versions prior to 4.8 (can't say for version 4.8).

**Comment by jneale** [comment]
Thanks for the detailed example and description. I can confirm this is an issue. I'll get this resolved as quickly as possible.

**Comment by jneale** [comment]
Fixed in 4.9
